src/components/common/SafeIcon/index.tsx

```diff
@@ -3,27 +3,36 @@ import { Box } from '@mui/material'
 
 import css from './styles.module.css'
 import Identicon, { type IdenticonProps } from '../Identicon'
+import classnames from 'classnames'
 
 interface ThresholdProps {
   threshold: number | string
   owners: number | string
 }
 const Threshold = ({ threshold, owners }: ThresholdProps): ReactElement => (
-  <Box className={css.threshold} sx={{ color: ({ palette }) => palette.static.main }}>
+  <Box className={classnames(css.badge, css.threshold)} sx={{ color: ({ palette }) => palette.static.main }}>
     {threshold}/{owners}
   </Box>
 )
 
+const PendingActionsBadge = ({ queuedTxs }: { queuedTxs: number }): ReactElement => (
```

Comment:
The badge isn't super clear IMHO.
I would experiment with adding a line of text under each Safe row, literally saying "N transactions need signatures".